backend/api/Constants/FileType.cs

```diff
@@ -10,5 +10,7 @@ public enum FileType
         Acquisition,
         [EnumMember(Value = "research")]
         Research,
+        [EnumMember(Value = "unknown")]
+        Unknown,
```

Comment:
I am guessing this is not really an unknown file type but a file type of type "unknown". If that's the case please add some comments so is not as confussing

Reply:
Yes, fair, will do.